dev-java/lombok-bin/lombok-bin-1.16.8.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+EAPI=6
+
+inherit eutils
+
+MY_PN="lombok"
+
+DESCRIPTION="Project Lombok makes java a spicier language by adding 'handlers' that know how to build and compile simple, boilerplate-free, not-quite-java code."
+HOMEPAGE="https://projectlombok.org/"
+SRC_URI="https://projectlombok.org/downloads/${MY_PN}-${PV}.jar"
+LICENSE="MIT"
+
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND=""
+RDEPEND=">=virtual/jdk-1.7
+	${DEPEND}"
```

Comment:
Repoman gave me a warning that there are multiple slots for the jdk so I also added :*
Is this correct?